test/unit/Services/UserService.test.ts

```diff
@@ -0,0 +1,57 @@
+import {expect} from "chai";
+import {createSandbox, SinonSandbox} from "sinon";
+import {UserService} from "../../../src/Services/UserService";
+import logger from "../../../src/Services/Logger";
+import {User} from "../../../src/Models/User";
+import sinon from "sinon";
+
+describe("UserService", function () {
+
+    let sandbox: SinonSandbox;
+    let userService: UserService;
+    let userFindStub: any;
+
+    beforeEach(function () {
+        sandbox = createSandbox();
+        userService = new UserService();
+        userFindStub = sinon.stub(User, "findOne");
+    });
+
+    afterEach(function () {
+        sandbox.restore();
+        logger.silent = false;
+        userFindStub.restore();
+    });
+
+    it("should be true provided email and password are valid", async () => {
```

Comment:
not much important but better to say "should successfully authenticate user" as with this title it is not sure which function you are testing from user service.